xrpl/models/transactions/permissioned_domain_set.py

```diff
@@ -0,0 +1,52 @@
+"""Model for PermissionedDomainSet transaction type."""
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.deposit_preauth import Credential
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+_MAX_ACCEPTED_CREDENTIALS_LENGTH = 10
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class PermissionedDomainSet(Transaction):
```

Comment:
ditto

Reply:
updated in [20762f8](https://github.com/XRPLF/xrpl-py/pull/773/commits/20762f8cbd2f438c28084c2735fbd9db6c8edfcc)